.github/dependabot.yml

```diff
@@ -19,8 +19,18 @@ updates:
       - "infrastructure"
       - "blocked"
 
-  # update our package.json everyday for the npm
+  # update our npm development dependencies monthly
   - package-ecosystem: "npm"
     directory: "/ui/webui"
     schedule:
-      interval: "daily"
+      interval: "monthly"
+    allow:
+        - dependency-type: "development"
```

Comment:
I guess this is the indentation error?

Reply:
Oh It's valid, but indeed I would use 2spaces for consistency.

Reply:
Updated.